data/addressbook.json

```diff
@@ -1,113 +1,3 @@
 {
-  "recipes" : [ {
-    "name" : "chickenn",
-    "time" : "15",
-    "isFavourite" : false,
-    "grains" : [ ],
-    "vegetables" : [ "vege,  20.0g" ],
-    "proteins" : [ ],
-    "fruits" : [ "kiwi,  50.0g" ],
-    "others" : [ "hello,  30.0g" ],
-    "steps" : [ "Testing output when duplicate steps are added during EDIT" ],
-    "goals" : [ "Bulk like the Hulk" ]
-  }, {
```

Comment:
I think you might have forgotten to delete this file and add, commit and push the recipebook.json file.
Additionally, ensure that there is a new line at the end of this .json file in order to pass the travis CI